packages/server/src/routes/calendar/event.dto.ts

```
@@ -0,0 +1,14 @@
+import { DBConnection } from "../../drizzle/drizzle-client";
+import { eventTable, InsertEvent } from "../../drizzle/schema";
+
+export class EventTable extends DBConnection {
+	constructor(TURSO_CONNECTION_URL: string, TURSO_AUTH_TOKEN: string) {
+		super(TURSO_CONNECTION_URL, TURSO_AUTH_TOKEN);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)